### DISCUSSION TITLE
F12 and Ctl-F12 with source generators in play ?

### DISCUSSION THREAD

**Discussion by stephenhauck** [CommunityToolkit/dotnet#594] [Q&A] [answered]
I LOVE the observable property feature and I feel silly asking this but when I am in code and want to jump to the definition using F12 it takes me to the partial class ... is there a way to got to my definition in code instead ?

**Comment by michael-hawker** [comment, marked as answer]
This is a limitation of how source generators work, but will be resolved by https://github.com/CommunityToolkit/dotnet/issues/555 when C# supports partial properties.
